### DISCUSSION TITLE
EV3 Bluetooth connection to visual code

### DISCUSSION THREAD

**Discussion by alpha71** [pybricks/support#349] [Q&A]
Hello, maybe the discussion was already pointed out but my question is if can connect EV3 to Visual Code via Bluetooth?

**Comment by dlech** [comment]
The [ev3dev extension](https://marketplace.visualstudio.com/items?itemName=ev3dev.ev3dev-browser) for VS Code supports this.

The [ev3dev docs](https://www.ev3dev.org/docs/tutorials/connecting-to-the-internet-via-bluetooth/) tell how to set up a network connection using Bluetooth.

On Windows, connecting with Bluetooth will give a timeout error since it doesn't support Internet Connection Sharing via Bluetooth, but it should still work after that.

**Reply by alpha71** [reply]
OK, so means it's a standard way to connection. Very well. Thank you again David.
